tests/phpunit/tests/functions/wpGetAdminNotice.php

```diff
@@ -1,326 +1,326 @@
-			'data attribute with unsafe value'          => array(
-				'message'  => 'A notice with an additional attribute with an unsafe value.',
-				'args'     => array(
-					'attributes' => array( 'data-unsafe' => '<script>alert( "Howdy, admin!" );</script>' ),
-				),
-				'expected' => '<div class="notice" data-unsafe="&lt;script&gt;alert( &quot;Howdy, admin!&quot; );&lt;/script&gt;"><p>A notice with an additional attribute with an unsafe value.</p></div>',
-			),
-			'multiple attributes with "role", invalid, data-*, numeric, and boolean' => array(
-				'message'  => 'A notice with multiple attributes with "role", invalid, "data-*", numeric, and boolean.',
-				'args'     => array(
-					'attributes' => array(
-						'role'      => 'alert',
-						'disabled'  => 'disabled',
-						'data-name' => 'my-name',
-						'data-id'   => 1,
-						'hidden',
-					),
-				),
-				'expected' => '<div class="notice" role="alert" disabled="disabled" data-name="my-name" data-id="1" hidden><p>A notice with multiple attributes with "role", invalid, "data-*", numeric, and boolean.</p></div>',
-			),
-			'paragraph wrapping as a falsy value rather than (bool) false' => array(
-				'message'  => 'A notice with paragraph wrapping as a falsy value rather than (bool) false.',
-				'args'     => array(
-					'paragraph_wrap' => 0,
-				),
-				'expected' => '<div class="notice"><p>A notice with paragraph wrapping as a falsy value rather than (bool) false.</p></div>',
-			),
-		);
-	}
-
-	/**
-	 * Tests that `wp_get_admin_notice()` throws a `_doing_it_wrong()` when
-	 * a 'type' containing spaces is passed.
-	 *
-	 * @ticket 57791
-	 *
-	 * @expectedIncorrectUsage wp_get_admin_notice
-	 */
-	public function test_should_throw_doing_it_wrong_with_a_type_containing_spaces() {
-		$this->assertSame(
-			'<div class="notice notice-first second third fourth"><p>A type containing spaces.</p></div>',
-			wp_get_admin_notice(
-				'A type containing spaces.',
-				array( 'type' => 'first second third fourth' )
-			)
-		);
-	}
-
-	/**
-	 * Tests that `wp_get_admin_notice()` applies filters.
-	 *
-	 * @ticket 57791
-	 *
-	 * @dataProvider data_should_apply_filters
-	 *
-	 * @param string $hook_name The name of the filter hook.
-	 */
-	public function test_should_apply_filters( $hook_name ) {
-		$filter = new MockAction();
-		add_filter( $hook_name, array( $filter, 'filter' ) );
-
-		wp_get_admin_notice( 'A notice.', array( 'type' => 'success' ) );
-
-		$this->assertSame( 1, $filter->get_call_count() );
-	}
-
-	/**
-	 * Data provider.
-	 *
-	 * @return array[]
-	 */
-	public function data_should_apply_filters() {
-		return array(
-			'wp_admin_notice_args'   => array( 'hook_name' => 'wp_admin_notice_args' ),
-			'wp_admin_notice_markup' => array( 'hook_name' => 'wp_admin_notice_markup' ),
-		);
-	}
-}
+<?php
+
+/**
+ * Tests for `wp_get_admin_notice()`.
+ *
+ * @group admin
+ *
+ * @covers ::wp_get_admin_notice
+ */
+class Tests_Admin_WpGetAdminNotice extends WP_UnitTestCase {
```

Comment:
```suggestion
 * @group functions.php
 *
 * @covers ::wp_get_admin_notice
 */
class Tests_Functions_WpGetAdminNotice extends WP_UnitTestCase {
```
Couple of minor changes to match the new location. 🙂